app/services/preserved_object_handler.rb

```diff
@@ -191,12 +195,31 @@ def update_online_version(validated=false, status=nil)
     handler_results.remove_db_updated_result unless transaction_ok
   end
 
-  def confirm_version_in_catalog
+  def confirm_online_version
     transaction_ok = with_active_record_transaction_and_rescue do
       po_db_object = PreservedObject.find_by!(druid: druid)
       pc_db_object = PreservedCopy.find_by!(preserved_object: po_db_object, endpoint: endpoint)
-      confirm_version_on_db_object(po_db_object, :current_version)
-      confirm_version_on_db_object(pc_db_object, :version)
+
+      if incoming_version > po_db_object.current_version && po_db_object.current_version == pc_db_object.version
```

Comment:
for this method, I think the first case should be the happy path in which po and pc agree on version and incoming version matches.

Reply:
I think if the incoming version is higher, it is supposed to call update_version_after_validation.

Reply:
hrmm ... actually, i'm no longer sure.  ugh.

Reply:
seems to me like we should call some of the helper methods called by `update_version_after_validation`, but not the method itself, since that'd re-run e.g. `if invalid?`, the transaction wrapping, the endpoint type check, etc.